SmartDeviceLink-iOS.podspec

```diff
@@ -290,7 +290,11 @@ s.public_header_files = [
 'SmartDeviceLink/SDLNotificationConstants.h',
 'SmartDeviceLink/SDLRequestHandler.h',
 'SmartDeviceLink/SDLRPCNotificationNotification.h',
-'SmartDeviceLink/SDLRPCResponseNotification.h'
+'SmartDeviceLink/SDLRPCResponseNotification.h',
+'SmartDeviceLink/SDLVideoStreamingFormat.h',
```

Comment:
In the v4.7 branch, the order of these should follow the `SmartDeviceLink.h` ordering. In the v5.0 branch they are all alphabetical.